repository-data/application/src/main/resources/hcm-config/configuration/modules/scxmlregistry.yaml

```diff
@@ -2,6 +2,14 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/scxmlregistry/hippo:moduleconfig/hipposcxml:definitions/documentworkflow:
+      /externalFilePublish:
+        hipposcxml:classname: uk.nhs.digital.externalstorage.workflow.action.ExternalFilePublishAction
```

Comment:
Under advice from @ziemsky we changed the package structure for these. Have a look at mine :slightly_smiling_face: